test/assertion.mbt

```diff
@@ -18,6 +18,20 @@ fn debug_string[T : Debug](t : T) -> String {
   buf.to_string()
 }
 
+/// Test that the two values are equal.
+///
+/// Returns Ok if the two arguments are equal by value, using `==`; raises an
+/// Error otherwise.
+///
+/// # Examples
+///
+/// ```moonbit
+/// let a = "4" + "2"
+/// let b = "4" + "2"
+/// @test.eq(a, a)?  // this is okay
```

Comment:
Using `@` in docstrings produces a compiler warning:

```
core/test/assertion.mbt:35:1-47:2 Warning 020: Invalid pragma, unexpected id or invalid properties
```

Is it a bug or feature?

Reply:
cc @Yoorkin

Reply:
> Using `@` in docstrings produces a compiler warning:
> 
> ```
> core/test/assertion.mbt:35:1-47:2 Warning 020: Invalid pragma, unexpected id or invalid properties
> ```
> 
> Is it a bug or feature?

It's a bug. The `@test.` should be ignored by the parser, but it's being recognized as pragmas. The pragmas is a new feature that we are planning to introduce today.